pom.xml

```diff
@@ -2619,6 +2619,25 @@
       </activation>
     </profile>
 
+    <profile>
+      <id>hadoop3.3</id>
+      <properties>
+        <hadoop.version>3.3.0</hadoop.version>
+      </properties>
+      <dependencies>
+        <dependency>
+          <groupId>org.codehaus.jackson</groupId>
+          <artifactId>jackson-core-asl</artifactId>
+          <version>1.9.13</version>
+        </dependency>
+      </dependencies>
```

Comment:
Do we support hadoop 3.x compile?

Reply:
We have been using -Dhadoop.version=3.3.0 for a long time and there have been no problems.

Reply:
We have a long suspended PR https://github.com/apache/hudi/pull/5786 before for Hadoop 3.x compatibility, not sure how it works for your env.

Reply:
PR https://github.com/apache/hudi/pull/5786  is adapted to the hadoop3.x environment, but if parquet.version is configured with version 0.12.2, the problem of missing jackson package will still occur. The currently submitted PR https://github.com/apache/hudi/pull/9726 is mainly to solve the problem of compilation failure caused by the lack of jackson dependency package in the Hadoop 3 environment. It is not adapted to the hadoop 3 environment.
As for adapting hadoop3, it may take longer to fully test and verify.

Reply:
Got it, jackson is a jar that is very easy to conflict.

Reply:
@Forus0322 Can you double check `jackson-core-asl` is included for hadoop3.3 profile only?